core/src/main/scala/astraea/spark/rasterframes/stats/CellHistogram.scala

```diff
@@ -49,6 +51,98 @@ case class CellHistogram(stats: CellStatistics, bins: Seq[CellHistogram.Bin]) {
 
     lines.mkString("\n")
   }
+
+  // find the count of the bin that the value fits into
+  def itemCount(label: Double): Long = {
+    // look at each pair of consecutive bins, and when one bin is <= the value
+    // and the other is > the value, return the smaller bin label
+    val sorted = bins.sortBy(_.value)
+    require(sorted.nonEmpty, "Sorted must be nonempty")
+    require(label >= labels.min, "Label must be within the range of the values")
+    val tBin = (0 until sorted.length - 1).find(i => sorted.apply(i).value <= label
```

Comment:
I _feel_ like this could be made more readable, but a suggestion isn't jumping out at me. Good job using functional API though.